src/main/kotlin/no/skatteetaten/aurora/boober/feature/FluentbitSidecarFeature.kt

```diff
@@ -0,0 +1,354 @@
+                name = splunkHostKey
+                valueFrom {
+                    secretKeyRef {
+                        key = splunkHostKey
+                        name = adc.hecSecretName
+                        optional = false
+                    }
+                }
+            },
+            newEnvVar {
+                name = splunkPortKey
+                valueFrom {
+                    secretKeyRef {
+                        key = splunkPortKey
+                        name = adc.hecSecretName
+                        optional = false
+                    }
+                }
+            }
+        )
+
+        return newContainer {
+            name = "${adc.name}-fluent-sidecar"
+            env = podEnvVariables.addIfNotNull(hecEnvVariables)
+            volumeMounts = listOf(
+                    newVolumeMount {
+                        name = adc.fluentParserName
+                        mountPath = parserMountPath
+                    }, newVolumeMount {
+                        name = adc.fluentConfigName
+                        mountPath = "/fluent-bit/etc"
+                    }, loggingMount
+            )
+            resources {
+                limits = mapOf(
+                        // TODO? Add as config parameter
+                        "memory" to Quantity("64Mi"),
+                        "cpu" to Quantity("100m")
+                )
+                requests = mapOf(
+                        "memory" to Quantity("20Mi"),
+                        "cpu" to Quantity("10m")
+                )
+            }
+            image = "fluent/fluent-bit:latest"
+        }
+    }
+}
+
+data class LoggingConfig(
+    val name: String,
+    val sourceType: String,
+    val index: String,
+    val filePattern: String
+)
+
+fun getLoggingIndexes(adc: AuroraDeploymentSpec, cmd: AuroraContextCommand, defaultIndex: String): List<LoggingConfig> {
+    val result: MutableList<LoggingConfig> = mutableListOf()
+    val loggers = cmd.applicationFiles.findSubKeys("logging/loggers").flatMap { key ->
+        val value: String = adc.get("logging/loggers/$key")
+        listOf(
+            LoggingConfig(key, getKnownSourceType(key), value, getKnownFilePattern(key))
+        )
+    }
+    result.addAll(loggers.filter { log -> !(log.index.equals("false")) })
+    // Add default loggers if they do not exist
+    val applicationLog = loggers.find { logger -> logger.name.equals(logApplication) }
+    if (applicationLog == null) {
+        val newApplicationLog = LoggingConfig(logApplication, getKnownSourceType(logApplication), defaultIndex, getKnownFilePattern(logApplication))
+        result.add(newApplicationLog)
+    }
+    val accessLog = loggers.find { logger -> logger.name.equals(logAccess) }
+    if (accessLog == null) {
+        val newAccessLog = LoggingConfig(logAccess, getKnownSourceType(logAccess), defaultIndex, getKnownFilePattern(logAccess))
+        result.add(newAccessLog)
+    }
+
+    // TODO Add custom loggers
```

Comment:
Skal dette gjøres eller ikke?

Reply:
Antar det, det er i kravspekken (som dog er et forslag), og gir jo fleksibilitet. Om det faktisk blir brukt er jeg mer usikker på, så jeg velger å levere første leveranse uten så kan vi eventuelt legge til om det skulle bli behov. Jeg kan fjerne kommentaren.

Reply:
Jeg er usikker på hvilk av de faste mønstrene som vi har brukt over her som er konfigurert i standard loggekonfigurasjon i baseimagene også. Jeg tror vi bør ta en ekstra gjennomgang av radish/architect og sørge for at logback konfigurasjon inneholder det man trenger her.